### PR TITLE
Feature add mcb cyclic example

### PR DESCRIPTION
**Pull request by jcastro-ingeniamc**

### CODE REVIEW

File: application/mcb-master-coc.c [hunk 1]
```diff
@@ -0,0 +1,151 @@
+/**
+ * @file mcb-master-cyclic.c
```
Comment (jcastro-ingeniamc) [inline application/mcb-master-coc.c:2]: mcb-master-coc.c

File: application/mcb-master-coc.c [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+ * @copyright Ingenia Motion Control (c) 2020. All rights reserved.
+ */
+
+#include "application.h"
+
+#include <string.h>
+#include "mcb.h"
+#include "mcb_al.h"
+
+#define MCB_NMB_INST    (uint16_t)1U
```
Comment (jcastro-ingeniamc) [inline application/mcb-master-coc.c:15]: doc defines

File: application/mcb-master-coc.c [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+/** Error codes */
+#define NO_ERROR                    (int16_t)0
+#define MCB_CURRENT_STATUS_ERROR    (int16_t)-1
+#define MCB_MAPPING_ERROR           (int16_t)-2
+
+#define MCB_TX_MAP_NMB              (uint16_t)2U
+#define MCB_RX_MAP_NMB              (uint16_t)2U
+
+int16_t
+SetMcb0CyclicMode(void);
```
Comment (jcastro-ingeniamc) [inline application/mcb-master-coc.c:27]: document function

File: application/mcb-master-coc.c [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+    memset((void*)tMcbMsg.u16Data, (uint16_t)0U, (MCB_MAX_DATA_SZ * sizeof(tMcbMsg.u16Data[(uint16_t)0U])));
+
+    /** Set mapping and move MCB to cyclic state */
+    int16_t i16CycSt = SetMcb0CyclicMode();
+
+    if (i16CycSt != NO_ERROR)
+    {
+        return -1;
+    }
+
```
Comment (jcastro-ingeniamc) [inline application/mcb-master-coc.c:60]: extra space

File: application/mcb-master-cyclic.c [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+ * @copyright Ingenia Motion Control (c) 2020. All rights reserved.
+ */
+
+#include "application.h"
+
+#include <string.h>
+#include "mcb.h"
+#include "mcb_al.h"
+
+#define MCB_NMB_INST    (uint16_t)1U
```
Comment (jcastro-ingeniamc) [inline application/mcb-master-cyclic.c:15]: doc defines and function header

File: application/mcb-master-cyclic.c [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+void AppStart(void)
+{
+    /** Set mapping and move MCB to cyclic state */
+    int16_t i16CycSt = SetMcb0CyclicMode();
+
+    if (i16CycSt != NO_ERROR)
+    {
+        return -1;
+    }
+
```
Comment (jcastro-ingeniamc) [inline application/mcb-master-cyclic.c:51]: extra space

File: application/mcb-master-coc.c [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+    tMcbMsg.u16Addr = 0x6E4;
+    tMcbMsg.eStatus = MCB_STANDBY;
+    memset((void*)tMcbMsg.u16Data, (uint16_t)0U, (MCB_MAX_DATA_SZ * sizeof(tMcbMsg.u16Data[(uint16_t)0U])));
+
+    /** Set mapping and move MCB to cyclic state */
+    int16_t i16CycSt = SetMcb0CyclicMode();
+
+    if (i16CycSt != NO_ERROR)
+    {
+        return -1;
```
Comment (ghost) [inline application/mcb-master-coc.c:58]: Use error code #defines or Enums instead of magic numbers

**Pull request merged**